plugin/trino-hive/src/main/java/io/trino/plugin/hive/parquet/ParquetReaderConfig.java

```diff
@@ -15,12 +15,17 @@
 
 import io.airlift.configuration.Config;
 import io.airlift.configuration.ConfigDescription;
+import io.airlift.configuration.DefunctConfig;
 import io.airlift.configuration.LegacyConfig;
 import io.airlift.units.DataSize;
 import io.trino.parquet.ParquetReaderOptions;
 
 import javax.validation.constraints.NotNull;
 
+@DefunctConfig({
+        "hive.parquet.fail-on-corrupted-statistics",
+        "parquet.fail-on-corrupted-statistics"
```

Comment:
add trailing comma so that new additions here don't need to change any existing lines (cleaner diff)